packages/seal-loupe-adapter/src/LoupeHelper.php

```diff
@@ -51,29 +51,41 @@ public function getLoupe(Index $index): Loupe
 
     public function existIndex(Index $index): bool
     {
-        return \file_exists($this->getIndexFile($index));
+        $indexDirectory = $this->getIndexDirectory($index);
+
+        return \file_exists($indexDirectory);
     }
 
     public function dropIndex(Index $index): void
     {
         if ($this->existIndex($index)) {
-            \unlink($this->getIndexFile($index));
-            \unlink($this->getConfigurationFile($index));
+            $indexDirectory = $this->getIndexDirectory($index);
+
+            // beside the .db and our own .loupe file there exists other files which we need to remove
```

Comment:
Why this complicated? Do you also need a directory to store configuration? I would recommend keeping the loupe directory and your own config directory separate (or have the loupe one inside yours).

Reply:
Changing the configuration (searchable fields, sortable fields, ...) is in `SEAL` a active manual process via a CLI Command. To achieve this in Loupe I'm dump the `Configuration` of Loupe into a File and that is used always, and so the configuration only get updated / index migrated to the new fields after the Command is run.

https://github.com/schranz-search/schranz-search/blob/37da7cc77928b56d7c047be20ded10e4dc7bd3b4/packages/seal-loupe-adapter/src/LoupeHelper.php#L75

Reply:
I didn't find another way here to remove a Index. As there is no buildin function in PHP removing a directory containing files. So I think the dropIndex will be still the same, when I introduce additonal directory I just need then do it also recursive.

Reply:
Okay, yes then it looks good. `symfony/filesystem` could do that but I understand you probably want to keep the deps low.

Reply:
Before the structure was for two index movies, news:

 - var/indexes/movies.db
 - var/indexes/movies.loupe
 - var/indexes/news.db
 - var/indexes/news.loupe
 
 Now it is:
 
 - var/indexes/movies/loupe.db
 - var/indexes/movies/... # other loupe files
 - var/indexes/movies/config.loupe
 - var/indexes/news/loupe.db
 - var/indexes/news/... # other loupe files
 - var/indexes/news/config.loupe

Reply:
Thx, yeah want currently avoid additional deps for such things :)